File: app/src/main/java/com/dicoding/moviecataloguerv/data/source/model/ContentRating.java
package com.dicoding.moviecataloguerv.data.source.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import lombok.Data;

@Data
public class ContentRating {
    @SerializedName("iso_3166_1")
    @Expose
    private String iso;

    @Expose
    private String rating;
}

File: app/src/main/kotlin/com/ryanrvldo/moviecatalogue/data/model/ContentRating.kt
package com.ryanrvldo.moviecatalogue.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ContentRating(
    @SerializedName("iso_3166_1") @Expose val iso: String,
    @Expose val rating: String
)